package_manifest.yaml

```diff
@@ -16,20 +16,11 @@ packages:
     copr_projects:
       - "{{ hostvars['pulpcore-copr'] }}"
     koji_tags:
```

Comment:
I don't think koji_tags is used at all anymore. Did we forget to clean this up previously?